pkg/datastore/example_multiinstance_test.go

```diff
@@ -38,54 +41,55 @@ func ExampleDataStore_multiInstance() {
 	DevInstanceCtx := instancer.WithInstanceId(ServiceAdminCtx, "Dev")
 	ProdInstanceCtx := instancer.WithInstanceId(ServiceAdminCtx, "Prod")
 
+	// Initializes the Datastore using the metadata authorizer and connection details obtained from the ENV variables.
 	ds, _ := datastore.FromEnv(datastore.GetCompLogger(), mdAuthorizer, instancer)
+
+	// Registers the necessary structs with their corresponding role mappings.
 	roleMapping := map[string]dbrole.DbRole{
 		SERVICE_AUDITOR: dbrole.READER,
 		SERVICE_ADMIN:   dbrole.WRITER,
 	}
-
-	if err := ds.Register(DevInstanceCtx, roleMapping, &Person{}); err != nil {
+	if err := ds.Register(context.TODO(), roleMapping, &Person{}); err != nil {
 		log.Fatalf("Failed to create DB tables: %+v", err)
 	}
 
-	// Insert with appropriate Dev Instance Ctx
+	// Inserts a record with a given Id (uId) using the context of the Dev instance.
 	rowsAffected, err := ds.Insert(DevInstanceCtx, p1)
 	fmt.Println(rowsAffected, err)
```

Comment:
@krishnamiriyala, fyi, if I run `ExampleDataStore_multiInstance` (as a unit test, from GoLand), then these `fmt.Println()` statements don't work and don't print anything